### ISSUE TITLE
FA-CORE-001 T014: Final audit — grep src/ for remaining pfa-v3-* localStorage references

### ISSUE THREAD

**Issue by araujobernardo** [araujobernardo/finance-analyser#696]
## Context

Part of **FA-CORE-001 — Migrate Core Transaction UI from localStorage to Database** ([plan](specs/030-migrate-transactions-db/plan.md)).

**Phase 7 — Polish**

## Task

Run a final audit of the entire `src/` directory for any remaining `pfa-v3-*` localStorage string literals that were not caught by earlier phases.

```bash
grep -r "pfa-v3-" src/
```

For each match found:
- Identify whether the code is still reading or writing a `pfa-v3-*` key
- Remove the read/write and replace with the appropriate AccountContext or API call
- If the code is dead (e.g. inside a deleted-but-not-removed block), delete it

If no matches are found, this task is a no-op — document that in the PR description.

## Acceptance Criteria

- [ ] `grep -r "pfa-v3-" src/` returns zero matches
- [ ] `npx tsc --noEmit` passes with no TypeScript errors
- [ ] All existing Vitest unit tests pass

## Files

- Any src/ file that still contains `pfa-v3-` string literals (fix or delete)

## Dependencies

- T009–T013 must all be merged first
